nights/2017-04-21-README.md

```diff
@@ -1,27 +1,36 @@
 # 2017-04-21
 
-## Reviewed by:   your_name_here
+## Reviewed by:   Adam Kline
 
 ## Raw image gallery: http://physics.mnstate.edu/feder_gallery/2017-04-21
 
 ## Unusual images?
 
 Are there any images that look unusual? List the file name of any unusual images for this night here, with description:
 
-+ `crazy-object-001R.fit` -- not sure what this is an image of
-+ `another-crazy-object-003B.fit` -- this looks more like a flat than a science image.
-+ `full-moon-005B.fit` -- nice satellite or plane track in here.
++ `calibration-013B.fit` -- Flat is not uniformly lit.
++ `calibration-014B.fit` -- Flat is not uniformly lit.
++ `calibration-015B.fit` -- Flat is not uniformly lit.
++ `calibration-016B.fit` -- Flat is not uniformly lit.
++ `calibration-017B.fit` -- Flat is not uniformly lit.
++ `calibration-018B.fit` -- Flat is not uniformly lit.
++ `calibration-019B.fit` -- Flat is not uniformly lit.
++ `calibration-020B.fit` -- Flat is not uniformly lit.
++ `calibration-021B.fit` -- Flat is not uniformly lit.
++ `calibration-022B.fit` -- Flat is not uniformly lit.
++ `artemis-002R.fit` -- Dust is not correctly divided from image. Issue continues through most R filter images.
++ `ey-uma-001I.fit` -- Satellite.
++ `ey-uma-003R.fit` -- Satellite.
```

Comment:
ey-uma-058R.fit -- faint satellite
m51-008R.fit -- satellite
Also, some of the 'flats' look fuzzy/unclear